hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/entity/sql/SqlEntityListener.java

```diff
@@ -410,6 +415,7 @@ public void onTopicMessage(TopicMessage topicMessage) throws ImporterException {
 
     @Override
     public void onTransaction(Transaction transaction) throws ImporterException {
+        transaction.setIndex(transactionIndex.incrementAndGet());
```

Comment:
Transaction index should start at zero.

```suggestion
        transaction.setIndex(transactionIndex.getAndIncrement());
```

Reply:
Updated